tests/cypress/e2e/unit_tests/p1_fleet.spec.ts

```diff
@@ -1277,3 +1277,48 @@ if (!/\/2\.7/.test(Cypress.env('rancher_version')) && !/\/2\.8/.test(Cypress.env
     )
   });
 }
+
+describe('Test move cluster to newly created workspace and deploy application to it.', { tags: '@p1'}, () => {
```

Comment:
Let's leave it here for now as is last test in `p1`. However, given that if the cluster is not properly moved back at the end of the test it may interfere with other tests, perhaps it would be wise to set at the end of RBAC or given that we have several tests that are a bit sensitive in this area, perhaps we could set them in separate spec.

Reply:
We will create new spec and I will move this test to that spec.(As discussed offline)